domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/casetype/CaseTypeService.java

```diff
@@ -12,6 +12,8 @@
 
     void createAll(JurisdictionEntity jurisdiction, Collection<CaseTypeEntity> caseTypes);
 
+    Optional<Boolean> findCaseTypeExists(String reference, String jurisdictionId);
```

Comment:
why we need to return a Optional<Boolean> rather that a Boolean? This is confusing. Optional are returned in an api when the method might not return any result, as an alternative to returning null. But this is not our case. The case either exists or not. Can we change to Boolean?

Reply:
done